modules/claims_api/app/workers/claims_api/claim_uploader.rb

```diff
@@ -24,7 +25,11 @@ def perform(uuid)
         uploader.retrieve_from_store!(claim_object.file_data['filename'])
         file_body = uploader.read
         ClaimsApi::Logger.log('526', claim_id: auto_claim.id, attachment_id: uuid)
-        service(auth_headers).upload(file_body, claim_upload_document(claim_object))
+        if Flipper.enabled? :lh_bd_claim_uploader
+          service(auth_headers).upload(claim: auto_claim, pdf_path: uploader.file.file)
```

Comment:
Should we log the response from this so we can ask about a BD `requestId` if we need to troubleshoot something?

Reply:
I'm becoming more and more of a fan of logging. I would vote we add something to the logging to clearly show that we're using the new benefits documents service. maybe similar to the `/search` method.

Reply:
I think the ID is OK since it's sequential, I don't think they can do a lookup by it, but when they lookup by `evss_id`, seeing that number match is a nice confidence boost.